arch/platform/srf06-cc26xx/platform.c

```diff
@@ -104,9 +105,14 @@ fade(unsigned char l)
 static void
 set_rf_params(void)
 {
-  uint16_t short_addr;
   uint8_t ext_addr[8];
 
+#if MAKE_MAC == MAKE_MAC_OTHER
```

Comment:
I think the BLEach MAC should be added as an "official" MAC in `Makefile.include` (as opposed to using `MAKE_MAC_OTHER`).